schema/cassandra/cadence/schema.cql

```diff
@@ -378,6 +379,7 @@ CREATE TABLE executions (
   };
 
 CREATE TABLE history_node (
+  created_time      timestamp,
```

Comment:
nit: add the new columns to the end of the list. by convention the columns here are used in `primary key` in same order